heaps/min_heap.py

```diff
@@ -11,68 +11,100 @@ def __repr__(self):
         return str(self.value)
 
 
-
 class MinHeap:
 
     def __init__(self):
         self.store = []
 
-
-    def add(self, key, value = None):
+    def add(self, key, value=None):
         """ This method adds a HeapNode instance to the heap
             If value == None the new node's value should be set to key
-            Time Complexity: ?
-            Space Complexity: ?
+            Time Complexity: O(log n)
+            Space Complexity: O(1)
         """
```

Comment:
👍  However space complexity is O(log n) because the heap_up function is recursive.